lib/permessage-deflate.js

```diff
@@ -345,7 +344,13 @@ class PerMessageDeflate {
         ? zlib.Z_DEFAULT_WINDOWBITS
         : this.params[key];
 
-      this._inflate = zlib.createInflateRaw({ windowBits });
+      this._inflate = zlib.createInflateRaw(
+        Object.assign(
+          {},
+          this._options.zlibOptions,
```

Comment:
I think we should use a different option object. For the `chunkSize ` could be different from the one used for the deflate stream.